dsl_parser/utils.py

```diff
@@ -32,6 +32,14 @@
                         constants,
                         exceptions)
 
+PY2 = sys.version_info[0] == 2
+PY3 = sys.version_info[0] == 3
+
+if PY3:
```

Comment:
Why is this needed?

Reply:
If this will ever become a python3 thing